app/components/common/BaseModal.vue

```diff
@@ -51,7 +51,10 @@
     .modal-header
       font-weight: bold
       font-size: 30px
+      position: relative
```

Comment:
Why did this need to be changed to `relative`? Was it to support the close button that is absolutely positioned?